### PR TITLE
添加主题MDM

### PR DESCRIPTION
**Pull request by TonyChenn**
一款MaterialDesign风格主题，轻巧，简单，干净。

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -2853,3 +2853,14 @@
...
+  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了
+  link: https://github.com/TonyChenn/mdm
+  preview: https://tonychenn.cn
+  tags:
+    - Material Design
+    - Light
+    - Faster
+    - clean
+    - Hitokoto
+    - Sample
```
Comment (curbengh) [inline source/_data/themes.yml]: you mean "simple"?

File: source/_data/themes.yml [hunk 2]
```diff
@@ -2853,3 +2853,14 @@
     - seo
     - 设计
     - literary
+- name: Mdm
+  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了
```
Comment (curbengh) [inline source/_data/themes.yml]: need english description

File: source/_data/themes.yml [hunk 3]
```diff
@@ -2853,3 +2853,14 @@
...
+  link: https://github.com/TonyChenn/mdm
+  preview: https://tonychenn.cn
+  tags:
+    - Material Design
+    - Light
+    - Faster
```
Comment (curbengh) [inline source/_data/themes.yml]: The "Light" here means bright background or "lightweight"? If it's lightweight, "fast" tag is not needed.

**Comment by TonyChenn** [comment]
Oh, I'm so sorry for the wrong words, and i will modify them soon, thank you . 




------------------ 原始邮件 ------------------
发件人: "curbengh"<notifications@github.com>; 
发送时间: 2019年10月26日(星期六) 晚上8:20
收件人: "hexojs/site"<site@noreply.github.com>; 
抄送: "852454151"<852454151@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [hexojs/site] 添加主题MDM (#1184)




@curbengh requested changes on this pull request.
 
 
In source/_data/themes.yml:
 > @@ -2853,3 +2853,14 @@      - seo      - 设计      - literary +- name: Mdm +  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了 +  link: https://github.com/TonyChenn/mdm +  preview: https://tonychenn.cn +  tags: +    - Material Design +    - Light +    - Faster +    - clean +    - Hitokoto +    - Sample  
you mean "simple"?
 
 
In source/_data/themes.yml:
 > @@ -2853,3 +2853,14 @@      - seo      - 设计      - literary +- name: Mdm +  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了  
need english description
 
 
In source/_data/themes.yml:
 > @@ -2853,3 +2853,14 @@      - seo      - 设计      - literary +- name: Mdm +  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了 +  link: https://github.com/TonyChenn/mdm +  preview: https://tonychenn.cn +  tags: +    - Material Design +    - Light +    - Faster  
The "Light" here means bright background or "lightweight"? If it's lightweight, "fast" tag is not needed.

File: source/_data/themes.yml (hunk 1 above)
Comment (TonyChenn) [reply source/_data/themes.yml]: lite

File: source/_data/themes.yml (hunk 2 above)
Comment (TonyChenn) [reply source/_data/themes.yml]: I have change to english description.

**Pull request merged**